### PR TITLE
Parser now working. Still missing AST generation.

### PR DESCRIPTION
**Pull request by humleflue**

### CODE REVIEW

File: CUP-parser/Lexer.jflex [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import java_cup.runtime.*;
+import java.util.*;
+
+%%
+%class Lexer
+%unicode
+%line
+%column
+%cup
+%standalone
```
Comment (itDaveDK) [inline CUP-parser/Lexer.jflex]: burde ikke være der

File: CUP-parser/Lexer.jflex [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    StringBuffer string = new StringBuffer();
+    
+    private Symbol symbol(int type) {
+        return new Symbol(type, yyline, yycolumn);
+    }
+    private Symbol symbol(int type, Object value) {
+        return new Symbol(type, yyline, yycolumn, value);
+    }
+
+    List<String> numbers = new ArrayList();
```
Comment (itDaveDK) [inline CUP-parser/Lexer.jflex]: slet

File: CUP-parser/Lexer.jflex [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+%}
+
+LineTerminator = \r|\n|\r\n
+InputCharacter = [^\r\n]
+WhiteSpace     = [ \t\f]|\r|\n|\r\n
```
Comment (itDaveDK) [inline CUP-parser/Lexer.jflex]: use line terminater var

File: CUP-parser/Lexer.jflex [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+Comment              = {TraditionalComment} | {EndOfLineComment}
+TraditionalComment   = "/*" [^*] ~"*/" | "/*" "*"+ "/"
+// Comment can be the last line of the file, without line terminator.
+EndOfLineComment     = "//" {InputCharacter}* {LineTerminator}?
+
+Identifier       = [:jletter:] [:jletterdigit:]*
+DecNumberLiteral = -?[0-9]+.[0-9]+|0|-?[1-9][0-9]*
+
+%eof{
+    System.out.print("Numbers: ");
```
Comment (itDaveDK) [inline CUP-parser/Lexer.jflex]: SLET

File: CUP-parser/Lexer.jflex [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+
+/* keywords */
+<YYINITIAL> "number"               { return symbol(sym.numberdcl); }
+
+<YYINITIAL> {
+    /* identifiers */ 
+    {Identifier}                   { return symbol(sym.id); }
+    
+    /* literals */
+    {DecNumberLiteral}             { numbers.add(yytext()); return symbol(sym.numberval); }
```
Comment (itDaveDK) [inline CUP-parser/Lexer.jflex]: slet numbers.ADD

**Comment by humleflue** [comment]
@itDaveDK https://github.com/humleflue/P4/pull/5/commits/4a0667bd975b03c9d6fd26cf2190e20ea110dddc and https://github.com/humleflue/P4/pull/5/commits/0d97f764e468f0efe552f13709aa3e24818efaee

**Review by itDaveDK — APPROVED** [review]

**Pull request merged**